GliaWidgets/Sources/View/Chat/GVA/QuickReply/QuickReplyView.swift

```diff
@@ -59,6 +59,18 @@ final class QuickReplyView: BaseView {
             collectionView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor)
         ])
     }
+
+    override func layoutSubviews() {
+        switch style.backgroundColor {
```

Comment:
Doesn't layoutSubviews also need super? @yurii-glia

Reply:
It does indeed

Reply:
fixed [5c4854f](https://github.com/salemove/ios-sdk-widgets/pull/715/commits/5c4854fa6272308185fc86bdb7928d659cf978fa)

Reply:
for consistency yes, however, afaik, Apple doc for `layoutSubviews` method mentions that it is not necessary to call super for this method.